src/plugins/blocksmith/components/pinterest-pin-block-modal/index.jsx

```diff
@@ -63,10 +63,20 @@ export default class PinterestPinBlockModal extends Component {
     this.handleChangeSelect = this.handleChangeSelect.bind(this);
   }
 
+  handleMouseOut = e => {
```

Comment:
@digitalartcrew use the same syntax for this method as the other class methods eg

```
  handleMouseOut(e) {
    e.stopPropagation();
  }
```

and then youll use it as this.handleMouseOut